DependencyInjection/Providers/GoogleProviderConfigurator.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace KnpU\OAuth2ClientBundle\DependencyInjection\Providers;
+
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+
+class GoogleProviderConfigurator implements ProviderConfiguratorInterface
+{
+    public function buildConfiguration(NodeBuilder $node)
+    {
+        // no custom options
```

Comment:
What about `hostedDomain` - I see that here: https://github.com/thephpleague/oauth2-google. It looks like there's also an `accessType`: https://github.com/thephpleague/oauth2-google/blob/master/src/Provider/Google.php#L20. I'm not familiar with either of these - I just spotted them on the driver :)